packages/block-library/src/table/style.scss

```diff
@@ -1,9 +1,7 @@
 .wp-block-table {
-	// Fixed layout toggle
-	&.has-fixed-layout thead,
-	&.has-fixed-layout tbody,
-	&.has-fixed-layout tfoot {
-		display: table;
+	overflow-x: auto;
```

Comment:
@jasmussen ah, I think this should be in theme.scss instead of style.scss, is that right?

Reply:
Fixed-width layout has to work everywhere because it's a user-set option.

But the table doesn't have to be responsive out of the box, no, so okay to remove the width, min-width, and overflow rules and put those in theme.scss 👍 👍